source/uwp/SharedRenderer/lib/WholeItemsPanel.cpp

```diff
@@ -104,8 +104,7 @@ namespace winrt::AdaptiveCards::Rendering::Xaml_Rendering::implementation
 
                                     // Text did not reach its minHeight property and we have to clear it otherwise
                                     // ellipses won't be displayed correctly...
-                                    winrt::IFrameworkElementStatics frameworkElementStatics;
-                                    child.ClearValue(frameworkElementStatics.MinHeightProperty());
+                                    child.ClearValue(winrt::FrameworkElement::MinHeightProperty());
```

Comment:
curious, have you searched MinHeightProperty() in other places if they use this syntax?

Reply:
great. just to be sure; I meant we searched & removed the old syntax.

Reply:
agreed, I did that before checking in -> no other instances found using the old syntax.